src/node-base64-image.js

```diff
@@ -47,6 +47,10 @@ export function encode(url: string, options: Object = {string: false, local: fal
         return callback(err);
       }
 
+      if (!response.headers['content-type'].match(/image/)) {
+          return callback(new Error('Error retrieving image - Not a Image'));
```

Comment:
Formatting is off here - should be 2 spaces instead of 4. And message should be `Error retrieving image - Not an Image`